src/halmos/__main__.py

```diff
@@ -192,6 +193,15 @@ def mk_block() -> Block:
 def mk_addr(name: str) -> Word:
     return BitVec(name, 160)
 
+def mk_caller(args: argparse.Namespace) -> Word:
+    if args.symbolic_msg_sender:
+        return mk_addr('msg_sender')
+    else:
+        return con_addr(999)
```

Comment:
not a big fan of 999 and 1000, for no really good reason other than they don't look like addresses 😅

I would prefer either of these:
- using well known values like the default msg.sender and test address used in foundry
- or if we don't want to couple tightly with foundry (their values may change, etc, etc), we could come up with new "well known values", e.g.:
```
➜ address(uint160(uint256(keccak256("halmos default msg.sender"))))
Type: address
└ Data: 0x6215e5fd0f1d85fb78fb8e27cede12699bfebb5a
➜ address(uint160(uint256(keccak256("halmos default address(this)"))))
Type: address
└ Data: 0x7d025cd0feaea4e87a75c716e3ed70279df723bd
```
(they're static but don't really look like anything in logs)
- or using "magic values" that pop in logs like the dolphin address (`0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE`)

Reply:
makes sense, but i'd like to have them to be smaller numbers since it makes it easier to assume they don't collide with other sha3 hash values generated at runtime. (we already have sha3 values are not very close to 0 or 2^256.)

would some hex values look like addresses?  like `0x1000` or `0xbeef0001`?